code/clue.js

```diff
@@ -1,59 +1,374 @@
+  lastName: 'Plum',
+  color: '#8B008B', 
+  description: 'He is a nuclear specialist',
+  age: 84,
+  image: 'assets/plum.png',
+  occupation: 'Retired',
+  favouriteWeapon: 'pistol' 
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: '#DC143C', 
+  description: 'She is a specialist in freelance espionage',
+  age: 28,
+  image: 'assets/scarlet.png',
+  occupation: 'Actress',
+  favouriteWeapon: 'knife'
+}
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: '#0000CD', 
+  description: 'She always maintains her dignity',
+  age: 57,
+  image: 'assets/peacock.png',
+  occupation: 'Socialite',
+  favouriteWeapon: 'candlestick' 
+}
+
+const colonelMustard = {
+  firstName: 'Colonel',
+  lastName: 'Mustard',
+  color: '#FCE38E', 
+  description: 'He is a great hunter',
+  age: 65,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel',
+  favouriteWeapon: 'axe' 
+}
 
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: '#FFFAF0	', 
+  description: 'Loyal and confidant',
+  age: 72,
+  image: 'assets/white.png',
+  occupation: 'Housekeeper',
+  favouriteWeapon: 'posion' 
+}
+
+//objects for weapons
 const rope = {
   name: 'rope',
-  weight: 10
+  weight: 10,
+  article: 'a',
+  id: 'rope',
+  foundIn: 'guestHouse'
+}
+
+const knife = {
+  name: 'knife',
+  weight: 7,
+  article: 'a',
+  id: 'knife',
+  foundIn: 'kitchen'
+}
+
+const candelstick = {
+  name: 'candelstick',
+  weight: 12,
+  article: 'a',
+  id: 'candelstick',
+  foundIn: 'hall'
+}
+
+const dumbbell = {
+  name: 'dumbbell',
+  weight: 24,
+  article: 'a',
+  id: 'dumbbell',
+  foundIn: 'spa'
+}
+
+const poison = {
+  name: 'poison',
+  weight: 1,
+  article: '',
+  id: 'poison',
+  foundIn: 'library'
+}
+
+const axe = {
+  name: 'axe',
+  weight: 17,
+  article: 'an',
+  id: 'axe',
+  foundIn: 'patio'
+}
+
+const bat = {
+  name: 'bat',
+  weight: 4,
+  article: 'a',
+  id: 'bat',
+  foundIn: 'billiardRoom'
+}
+
+const trophy = {
+  name: 'trophy',
+  weight: 29,
+  article: 'a',
+  id: 'trophy',
+  foundIn: 'livingRoom'
+}
+
+const pistol = {
+  name: 'pistol',
+  weight: 11,
+  article: 'a',
+  id: 'pistol',
+  foundIn: 'lounge'
+}
+
+//objects for rooms
+const diningRoom = {
+  name: 'dining Room',
+  id: 'dining room'
+}
+
+const conservatory = {
+  name: 'conservatory',
+  id: 'conservatory'
+}
+
+const kitchen = {
+  name: 'kitchen',
+  id: 'kitchen'
+}
+
+const study = {
+  name: 'study',
+  id: 'study'
+}
+
+const library = {
+  name: 'library',
+  id: 'library'
+}
+
+const billiardRoom = {
+  name: 'billiard Room',
+  id: 'billiard Room'
+}
+
+const lounge = {
+  name: 'lounge',
+  id: 'lounge'
+}
+
+const ballroom = {
+  name: 'ballroom',
+  id: 'ballroom'
+}
+
+const hall = {
+  name: 'hall',
+  id: 'hall'
+}
+
+const spa = {
+  name: 'spa',
+  id: 'spa'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const livingRoom = {
+  name: 'living Room',
+  id: 'living Room'
```

Comment:
id's should have single words connected by `-`, so :
```
id: 'living-room'
```

Reply:
To be consistent throughout the code, I updated room IDs with kebabCase. Don't know the "best" approach for IDs, have gotten different answer depending on who I ask. Does it matter? (of course if I'm consistent).